src/utils/generalVariables.js

```diff
@@ -2,20 +2,21 @@
 // When running on a local machine, the URL will default to REACT_APP_API_URL - provided the app is run using `yarn start`
 // When hosted on Netlify, it'll retrieve the requisite environmental URL and target the appropriate API
 export const URL = process.env.REACT_APP_API_URL || "https://tieme-ndo-backend-staging1.herokuapp.com";
+const API_VERSION = '/api/v1';
```

Comment:
Could you change it so that it doesn't include the starting forward slash? Would make for clearer URL overviews down below.